src/HotChocolate/Stitching/test/Stitching.Types.Tests/Pipeline/__snapshots__/ApplyRemoveMiddlewareTests.Apply_FieldRemove.snap

```diff
@@ -0,0 +1,11 @@
+﻿extend schema @_hc_service(name: "abc") {
+
+}
+
+type Foo {
```

Comment:
the types should be removed if they are fully empty.

Reply:
I've submitted a separate PR for the cleanup middleware here: https://github.com/ChilliCream/hotchocolate/pull/5174 should I combine the two? With these two processes separated, it does mean that the remove middleware can leave the schema in an inconsistent state...